sprimber-classic/src/main/java/com/griddynamics/qa/sprimber/discovery/ClassicAdapterConstants.java

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) 2010-2018 Grid Dynamics International, Inc. All Rights Reserved
+http://www.griddynamics.com
+
+This library is free software; you can redistribute it and/or modify it under the terms of
+the GNU Lesser General Public License as published by the Free Software Foundation; either
+version 2.1 of the License, or any later version.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+$Id: 
+@Project:     Sprimber
+@Description: Framework that provide bdd engine and bridges for most popular BDD frameworks
+*/
+
+package com.griddynamics.qa.sprimber.discovery;
+
+/**
+ * @author fparamonov
+ */
+public abstract class ClassicAdapterConstants {
+
+    public static final String CLASSIC_ADAPTER_NAME = "classic";
+    public static final String CLASSIC_SUITE_ROLE = "testSuite";
+    public static final String CLASSIC_TEST_CASE_ROLE = "testCase";
+    public static final String CLASSIC_TEST_ROLE = "test";
+    public static final String CLASSIC_STEP_ROLE = "step";
```

Comment:
Don't we have some before/after for classic tests?

Reply:
not exactly, since the classic tests assume plain code invocation it is possible to implement before after test as part of code for the test. more global before and after may be added later.